wiki/Rules/el.md

```diff
@@ -0,0 +1,79 @@
+
+1.  Κάθε παίκτης δικαιούται να δημιουργήσει, να παίξει και να έχει προσβασιμότητα μόνο σε ΈΝΑΝ λογαριασμό. Η δημιουργία πολλαπλών λογαριασμών ή η κοινή χρήση λογαριασμών με άλλους χρήστες θα οδηγήσει σε σοβαρές ποινές.
+2.  Να είστε ευγενικοί. Η παρενόχληση ή άλλου είδους διαμάχης δεν έχει καμία θέση στη κοινότητα του osu!. Η συνεχή παραβίαση αυτού του κανονισμού μπορεί να οδηγήσει στην μόνιμη απώλεια της ικανότητας να επικοινωνείτε με άλλους χρήστες.
+3.  Μην συμπεριφέρεσαι σαν κόπανος. Εκεί που δεν υπερτερούν οι κανόνες, θα πρέπει να υπερτερήσει η κοινή λογική. Οι διαχειριστές έχουν την διακριτική ευχέρεια να εκφράσουν την γνώμη τους επί του θέματος εάν κριθεί απαράιτητο.
+4.  Η υπερβολική χυδαιότητα δεν είναι ευπρόσδεκτη. Οι βρισιές ή οι βλασφημίες επιτρέπεται να χρησιμοποιούνται, αλλά όχι αν αρχίζουν να παραβιάζουν τους κανόνες \#2 και \#3.
+5.  Η κοινότητά μας είναι για όλες τις ηλικίες. Αυτό σημαίνει ότι περιεχόμενο 18+/NSFW όπως η χρήση ναρκωτικών ή απροκάλυπτα σεξουαλικά θέματα δεν είναι ευπρόσδεκτα εδώ. Εξαιρέσεις γίνονται ανάλογα με την περίπτωση, αλλά σε γενικές γραμμές, κρατήστε το καθαρό.
+
+Forum Κανόνες
+-------------
+
+1.  Μην κάνετε spam και μη γεμίζετε τα forum με άχρηστες αναρτήσεις/περιεχόμενα. Εάν δεν συμβάλλετε ουσιαστικά σε μια συζήτηση, καλύτερα να μη σχολιάσετε καθόλου.
+2.  Μπορείτε να στείλετε link με NSFW (ακατάλληλο) υλικό ΜΟΝΟ εάν είναι σκόπιμο σε μια συζήτηση, και μόνο αν το έχετε επισημάνει ως NSFW, και δεν είναι ενσωματωμένο σε ένα post.
+3.  Απαγορεύονται οι προσωπικές επιθέσεις. Οι κριτικές όλων των ειδών είναι ευπρόσδεκτες και ενθαρρύνονται, αλλά οι επιθέσεις προσωπικά σε άλλους χρήστες, όχι. Εάν δεν μπορείτε να αποδήξετε το επιχείρημά σας χωρίς τέτοιες τακτικές, καλύτερα να μην σχολιάσετε καθόλου.
+4.  Απαγορεύεται να κάνετε post σε πολύ παλιά και αδρανή threads, χωρίς περιεχόμενο στο post σας. Τέτοια πράξη μπορεί να οδηγήσει σε αυστηρή τιμωρία, αν γίνει επανειλημμένα.
+5.  Η αποστολή κακόβουλων συνδέσμων malware/adware/spyware/λογισμικών με ψευδή πρόθεση ή περιγραφή απαγορεύεται. Σε περίπτωση αμφιβολίας, ρωτήστε έναν συντονιστή (moderator).
+6.  Τα signatures δεν θα πρέπει να υπερβαίνουν τα 265kb στο συνολικό μέγεθος αρχείου. Οι κινούμενες εικόνες επιτρέπονται, εφ 'όσον είναι λογικές και δεν περιέχουν φωτοευαίσθητα στοιχεία (γρήγορη αναλαμπή κ.λπ.).
+7.  Τα κινούμενα avatar επιτρέπονται, αλλά υπο τους ίδιους όρους που ισχύουν και τα signatures. Τα avatar πρέπει πάντα να είναι κατάλληλα για όλες τις ηλικίες.
+8.  Οι διαφημίσεις και οι σύνδεσμοι (links) με σκοπό την διαφήμιση απαγορεύονται ρητά.
+9.  Περιεχόμενα που φορτώνουν αυτόματα (youtube βίντεο, εφαρμογές flash κ.λπ.) δεν επιτρέπονται.
+
+BSS Forum Κανόνες
+-----------------------------------
+
+Aυτοί οι κανόνες ισχύουν μόνο για τα [BSS](/wiki/Glossary) ([Beatmap](/wiki/Glossary) Submission System) threads. Αυτά τα threads δημιουργούνται αυτόματα μέσω του osu! client όταν ένα map υποβάλλεται μέσω του [editor](/wiki/Beatmap_Editor).
```

Comment:
I wrote this the same way the original is written. Unless the original has been changed, this should be fine.